kernel/aster-nix/src/fs/path/mount.rs

```diff
@@ -190,6 +191,43 @@ impl MountNode {
         Ok(())
     }
 
+    /// Move the process root and cwd to the new mount node tree.
```

Comment:
Where is the new mount node tree?  After I finished reading the code, I realized that "the new mount node tree" actually refers to "self," so the comment here is not clearly explained.

Reply:
self is the root of the mount node tree that needs to be moved. Before using this method, we often need to copy a new mount node tree. I have improved the comments of this method.